packages/corejs/src/inspections/index.ts

```diff
@@ -75,25 +75,30 @@ export const getMany = async ({ params, ...requestConfig }) => {
  * @param {[Object]} data.damageAreas - damageArea entity
  * @param {Object} requestConfig
  */
-export const upsertOne = async ({ data, ...requestConfig }) => {
-  const axiosResponse = await axios.request({
-    ...config.axiosConfig,
-    method: 'post',
-    url: `/inspections`,
-    data: mapKeysDeep(data, (v, k) => snakeCase(k)),
-    ...requestConfig,
-  });
-
-  const inspection = axiosResponse.data;
-  const id = inspection[idAttribute];
-  const createdAt = inspection.createdAt || Date.now();
-  const entity = { ...data, [idAttribute]: id, createdAt };
+export const upsertOne = async ({ data, ...requestConfig }: AxiosRequestConfig) => {
+  try {
+    const axiosResponse = await axios.request({
+      ...config.axiosConfig,
+      method: 'post',
+      url: `/inspections`,
+      data: mapKeysDeep(data, (v, k) => snakeCase(k)),
+      ...requestConfig,
+    });
+    const inspection = axiosResponse.data;
+    const id = inspection[idAttribute];
+    const createdAt = inspection.createdAt || Date.now();
+    const entity = { ...data, [idAttribute]: id, createdAt };
+
+    return ({
+      axiosResponse,
+      [idAttribute]: id,
+      ...normalize(entity, schema),
+    });
+  } catch(err) {
+    console.log('Error found !!');
+    console.log(err);
```

Comment:
We should let the developer catch his.her error him.herself or `throw err;` in our own catch.